static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -97,7 +97,12 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 		var self = this;
 		var resultObject = $trigger.parents(self.options.containerSelector).data('resultObject');
 		var metadata = resultObject.metadata;
-		
+
+		var fullUrl = document.location.href;
+		var baseUrl = URLUtilities.getBaseUrl(document.location.origin, true);
```

Comment:
just curious because i don't understand this fully: why are you using document.location.origin here instead of the value of fullUrl?